.github/workflows/release.yml

```diff
@@ -6,6 +6,14 @@ on:
       - v*
 
 jobs:
+  notify-runner:
+    steps:
+      - name: Notify Runner
+        uses: peter-evans/repository-dispatch@v1.0.0
+        with:
+          token: ${{ secrets.TOKEN }}
```

Comment:
As discussed, we need to make this TOKEN value using a service account.

Reply:
We should make this specific and call it `REPO_ACCESS_TOKEN`

Reply:
Someone needs to add this `REPO_ACCESS_TOKEN` now, done!